### PR TITLE
testing/glide: new aport

### PR DESCRIPTION
**Pull request by leprechau**
Package Management for Go

https://glide.sh

### CODE REVIEW

**Comment by Barthalion** [comment]
Uh, just no. I know how ungrateful and tedious job is packaging Go software, but we won't put pre-baked binary as an official package in aports.

**Comment by leprechau** [comment]
Fair enough. I can build it.

-- Aaron

**Comment by leprechau** [comment]
@Barthalion does this work?

File: testing/glide/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+url="https://github.com/Masterminds/glide"
+arch="all"
+license="MIT"
+depends=""
+makedepends="go"
+install=""
+subpackages=""
+source="$pkgname-$pkgver.tar.gz::https://github.com/Masterminds/glide/archive/$pkgver.tar.gz"
+builddir="$srcdir/go/src/github.com/Masterminds/glide"
+options="!strip"
```
Comment (Barthalion) [inline testing/glide/APKBUILD]: Is there any reason not to strip the binary? I always thought it is possible without causing issues.

Comment (leprechau) [inline testing/glide/APKBUILD]: It's not supported or recommended to strip go binaries with the regular ```strip``` implementation.  You can pass some ```ldflags``` to the build though that have a similar effect.

**Comment by Barthalion** [comment]
Merged in 0dbfd788.

**Pull request closed without merging**